android/expoview/src/main/java/host/exp/exponent/kernel/services/ExpoKernelServiceRegistry.java

```diff
@@ -84,6 +84,9 @@ public PermissionsKernelService getPermissionsKernelService() {
     return mPermissionsKernelService;
   }
 
+  /**
+   * Remove once SDK38 is phased out.
```

Comment:
all the files in this package (including ReactNativeActivity, ExperienceActivity, etc.) are not versioned and will therefore need to work for all supported SDK versions, including 38 and below. Is that going to be a problem since it seems like we are not calling into the SplashScreenKernelService anymore?

Reply:
I'm not sure what your concern is here 🤔 
`SplashScreenKernelService` is now serving as a simple forwarder to the `expo-splash-screen` module and is only used by the old versioned SDKs (including SDK 38).
`expo-splash-screen` module is always used in it's newest version as it's being referenced from unversioned code.

Reply:
As long as the SplashScreen module works for all older SDK versions then that addresses my concerns 👍